graph/src/data/subgraph/schema.rs

```diff
@@ -136,6 +137,15 @@ impl DeploymentCreate {
         self.debug_fork = fork;
         self
     }
+
+    pub fn entities_with_causality_region(
```

Comment:
offchain_entities?

Reply:
An entity table might have a causality region column and stores some entities from onchain data sources. So calling this `offchain_entities` might be more confusing.